Question about embedding deps

Does discord have other go projects that rely on lilliput internally and if so how are the deps being handled? The reason I ask is because I find myself having build scripts outside of the usual go process I am familiar with to download the linux or osx static libraries. I have recently been thinking of making a dummy embed file so that go always pulls the folder down as can be seen here: https://github.com/discord/lilliput/compare/master...Waldeedle:lilliput:feature/embed-deps 

Appreciate any insights 🙏

![image](https://github.com/user-attachments/assets/13a810a4-77d1-46ca-bc84-207597f0224f)
running the example app provided in the lilliput repo, is a quick way to reproduce this issue.
```@Waldeedle ➜ /workspaces/lilliput (fix/single-frame-from-animated-images) $ cd examples
@Waldeedle ➜ /workspaces/lilliput/examples (fix/single-frame-from-animated-images) $ go mod vendor
@Waldeedle ➜ /workspaces/lilliput/examples (fix/single-frame-from-animated-images) $ go run main.go
# github.com/discord/lilliput
In file included from ./avcodec.hpp:4,
                 from vendor/github.com/discord/lilliput/avcodec.go:3:
./opencv.hpp:7:10: fatal error: opencv2/core/fast_math.hpp: No such file or directory
    7 | #include <opencv2/core/fast_math.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.```
If you add the fake embed file I made, it recognizes those folders and their respective files as part of the code to be distributed.

--- a/embed.go
+++ b/embed.go
@@ -1,0 +1,11 @@
+//go:build EMBED_DEPS && darwin
+// +build EMBED_DEPS,darwin
+
+package lilliput
+
+import "embed"
+
+//go:embed deps/linux/*
+//go:embed deps/osx/*
+//go:embed icc_profiles/*
+var  embed.FS

--- a/go.mod
+++ b/go.mod
@@ -1,3 +1,3 @@
 module github.com/discord/lilliput
 
-go 1.15
+go 1.16

